cloudify_types/cloudify_types/component/utils.py

```diff
@@ -133,3 +134,21 @@ def blueprint_id_exists(client, blueprint_id):
 def deployment_id_exists(client, deployment_id):
     deployment = get_deployment_by_id(client, deployment_id)
     return True if deployment else False
+
+
+def should_upload_plugin(plugin_yaml_path, existing_plugins):
+    with open(plugin_yaml_path, 'r') as plugin_yaml_file:
+        plugin_yaml = yaml.load(plugin_yaml_file)
+        plugins = plugin_yaml.get('plugins')
+        for _, plugin_info in plugins.iteritems():
+            package_name = plugin_info.get('package_name')
+            package_version = str(plugin_info.get('package_version'))
```

Comment:
why str()? Combined with get, this might end up being the string "None", is that what we want?

Reply:
Because of that, the parser can convert the version into a float by accident